packages/components/Input/UserSearchInput.js

```diff
@@ -93,12 +94,25 @@ const createNewMember = email => ({
   isNew: true,
 });
 
-class UserSearchInputComponent extends Component {
-  state = this.prepareInitialState();
-
-  getPostfix() {
-    const { members, filterText, isLoading } = this.state;
+function UserSearchInputComponent({ blackList, onClickAdd, className }) {
```

Comment:
There are a little bit different kind of Component function. 
I mean use const instead of function

Reply:
Updated